pygit2/__init__.py

```diff
@@ -28,6 +28,9 @@
 # Import from the future
 from __future__ import absolute_import
 
+# Import from core python modules
+from ssl import get_default_verify_paths as default_ssl_verify_paths
```

Comment:
Please don't rename this callable. It's an action, not a constant. So I must start with a verb. `get`. Renaming it to constant/var-style is harmful and misleading for readability.
```suggestion
from ssl import get_default_verify_paths
```